### DISCUSSION TITLE
What would be the best way to run DF-aggregator continuously?

### DISCUSSION THREAD

**Discussion by balabanov** [ckoval7/df-aggregator#7] [Q&A]
I'm thinking of setting up a cloud DF-aggregator instance with a set of mobile receivers that might pop on and off on their own.

Besides maintaining the DB size, are there other considerations for 24/7 configuration? Thanks!

**Comment by ckoval7** [comment]
I've never run it much longer than 24 hours myself, but I can't imagine it would be a problem. Definitely keep an eye on the DB size, and maybe consider automatically rotating it with a cron job. Also, keep security in mind. This was only ever meant to run on private networks, and any remote access should be done over a VPN.

**Comment by balabanov** [comment]
Would you use `screen` to keep it running or some combo of nginx+UWSGI?
